<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>afterGameDrawDialog_2</class>
 <widget class="QDialog" name="afterGameDrawDialog_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>421</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>赛后新闻发布会</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>581</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>记者</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>541</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>就目前球队的状态而言，你有什么想说的？</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>691</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>等线</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>您</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>641</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
     </font>
    </property>
    <property name="text">
     <string>在这场比赛中，球员的状态毫无疑问并没有达到最佳。</string>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>641</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
     </font>
    </property>
    <property name="text">
     <string>虽然没有赢得胜利，但球员已经付出了最大的努力。</string>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>641</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
     </font>
    </property>
    <property name="text">
     <string>球员的状态令我失望，我认为他们应该在训练场上花费更多的精力。</string>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
